tool/site/client/src/components/claim/createDispenseClaim.ts

```diff
@@ -52,7 +52,11 @@ export function createClaim(
   const finalMedicationDispense = medicationDispenses[medicationDispenses.length - 1]
   const prescriptionStatusExtension = getTaskBusinessStatusExtension(finalMedicationDispense.extension)
 
-  const contained = [tempPractitionerRole]
+  // TODO when dispensing changes come though - pull from here
+  // const containedPractitionerRole = medicationDispenses[0].contained
+  //   ?.find(resource => resource?.resourceType === "PractitionerRole") as fhir.PractitionerRole
```

Comment:
We've done the dispensing changes so I think we shouldbe good to pull both `practitionerRole` and `organization` from the medication dispense

Reply:
done

Reply:
id be careful, the `organization` doesnt live in `contained` for dispense-notification, its a resource at the bundle level. i think we will have to take another look at this